### ISSUE TITLE
Multiple SPI device support

### ISSUE THREAD

**Issue by saawsm** [tuupola/hagl_esp_mipi#39]
It seems multiple SPI devices on the same bus aren't supported, as the bus is acquired without a way of releasing it.

https://github.com/tuupola/hagl_esp_mipi/blob/4630e89d7ab1eaee9645fe187f3237d4fcb7acbd/src/mipi_display.c#L283

[mipi_display_close()](https://github.com/tuupola/hagl_esp_mipi/blob/4630e89d7ab1eaee9645fe187f3237d4fcb7acbd/src/mipi_display.c#L320) has the corresponding release function, but isn't usable since the [spi_device_handle](https://github.com/tuupola/hagl_esp_mipi/blob/4630e89d7ab1eaee9645fe187f3237d4fcb7acbd/src/hagl_hal_double.c#L64) is static.

Is this something that is planned, or is it already supported, and I'm missing something?

**Comment by dizcza** [comment]
@saawsm I see you've been trying to work around the multiple SPI bus issue in your fork. Have you solved it?

**Comment by saawsm** [comment]
@dizcza Yeah, I ended up adding a `MIPI_DISPLAY_EXCLUSIVE_BUS` option that, if false, will release the bus after each display flush.

https://github.com/saawsm/hagl_esp_mipi/commit/cdc5bd489e4fde86d514761deca7ee344d76e4c2
https://github.com/saawsm/hagl/commit/eb5b48e3a3d41b36f0b138ea7c58f555c6fb6738
